wp-content/themes/twentyseventeen-wp20/header.php

```diff
@@ -27,7 +27,8 @@
 						<div class="wrap wrap-wide">
 							<?php get_template_part( 'template-parts/header/navigation', 'top' ); ?>
 							<?php Locales\locale_switcher(); ?>
-							<div class="navigation-top-menu-container" data-placeholder="<?php esc_attr_e( 'Search languages...', 'wp20' ); ?>"></div>
+							<!-- data-placeholder="<?php esc_attr_e( 'Search languages...', 'wp20' ); ?>" -->
```

Comment:
Seems like this isn't required with the `label .screen-reader-text` and [placeholder in JS](https://github.com/WordPress/wp20.wordpress.net/pull/93/files#diff-c86a86024c4de465cc9785b050a285506c9b64f456cc4c7b923c48345cfc021eR85)